src/forscape_static_pass.cpp

```diff
@@ -1732,7 +1745,16 @@ ParseNode StaticPass::resolvePower(ParseNode pn){
             //return ast.setComplement(base);
 
         case OP_MAYBE_TRANSPOSE:
-            warnings.push_back(Error(parse_tree.getSelection(rhs), TRANSPOSE_T));
+            switch (Program::instance()->settings.warningLevel<WARN_TRANSPOSE_T>()) {
```

Comment:
Settings should be accessed directly. Use of the program singleton violates the law of Demeter, and will bite you if there are ever multiple programs allowed